portfolio/src/main/webapp/style.css

```diff
@@ -1,65 +1,263 @@
-#menu {
-    color: white;
+#home-page {
+    font-family: "Lato", sans-serif;
+    /*background-image: url("/images/bg.jpg");*/
+}
+
+.home-image {
+    bottom: 0px;
+    height: 100%;
+    left: 800px;
+    position: absolute;
+    top: 0px;
+    width: 700px;
+}
+
+.intro {
+    font-size: 170%;
+    left: 700px;
     position: relative;
-    text-align: center;
+    text-align: justify;
+    top: 160px;
+    width: 500px;
 }
 
-#menuimage {
-    height: 800px;
-    width: 100%;
+.top-cloud {
+    height: 320px;
+    position: absolute;
+    top: 30px;
+    width: 650px;    
 }
 
-#name {
-    color: white;
-    font-size: 600%;
-    left: 60px;
+#bottom-cloud {
+    height: 320px;
     position: absolute;
-    text-shadow: 2px 2px 5px black;
+    top: 360px;
+    width: 650px;
+}
+
+#sub-page {
+    font-family: "Lato", sans-serif;
+    font-weight: 100px;
+}
+
+.slideshow-image {
+    vertical-align: middle;
+    width:100%;
     top: 100px;
 }
 
-#fact-button {
-    background-color: turquoise;
+/* On the side navigation bar */
+.sidenav {
+    background-color: #111;
+    height: 100%;
+    left: 0;
+    overflow-x: hidden;
+    padding-top: 250px;
+    position: fixed;
+    top: 0;
+    width: 200px;
+    z-index: 1;
+}
+
+.sidenav a {
+    color: #818181;
+    display: block;
+    font-size: 25px;
+    padding: 6px 6px 6px 32px;
+    text-decoration: none;
+}
+
+.sidenav a:hover {
+    color: #f1f1f1;
+}
+
+/* Applies to the text outside the side bar */
+.main {
+    margin-left: 200px; /* Same as the width of the sidenav */
+}
+
+/* On the name from the home page */
+.name {
+    font-size: 70px;
+    position: relative;
+    text-align: center;
+    top: 150px;
+}
+
+/* On the side bar */
+@media screen and (max-height: 450px) {
+  .sidenav {
+      padding-top: 15px;
+    }
+  .sidenav a {
+      font-size: 18px;
+    }
+}
+
+.fact-button {
+    align-items: center;
+    background-color: black;
     border-radius: 2em;
     box-sizing: border-box;
     color: white;
     display: inline-block;
-    font-size: 150%;
-    font-weight: 300;
-    left: 60px;
+    font-size: 100%;
+    font-weight: 200;
+    left: 850px;
     margin: 0 0.3em 0.3em 0;
     padding: 0.3em 1.2em;
-    position: absolute;
+    position: relative;
     text-align: center;
     text-shadow: 2px 2px 5px black;
-    top: 350px;
+    top: 200px;
     transition: all 0.2s;
 }
 
+.gallery {
+    color: white;
+    font-size: 300%;
+    left: 450px;
+    position: absolute;
+    text-decoration: none;
+    top: 160px;
+}
+
+.gallery:hover {
+    color:white;
+    text-shadow: 2px 2px 5px black;
+}
+
+#talent {
+    top:500px;
```

Comment:
nit: here, and in a few other places you have no whitespace after the colon `:`.